packages/model-viewer/src/features/post-processing.ts

```diff
@@ -0,0 +1,46 @@
+/* @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {property} from 'lit/decorators.js';
+import {Texture, Vector2} from 'three';
+
+import ModelViewerElementBase, {$needsRender, $renderer} from '../model-viewer-base.js';
+import {clamp, Constructor, deserializeUrl} from '../utilities.js';
+
+export declare interface PostProcessingInterface {
+  bloomEffect: boolean;
+  vignetteEffect: boolean;
+  ssaoEffect: boolean;
+}
+
+export const PostProcessingMixin = <T extends Constructor<ModelViewerElementBase>>(
+    ModelViewerElement: T): Constructor<PostProcessingInterface>&T => {
+  class PostProcessingModelViewerElement extends ModelViewerElement {
+    @property({type: Boolean, attribute: 'bloom-effect'})
```

Comment:
Starting a thread here for the bloom on a transparent background issue. That's a long thread you linked to: can you summarize and maybe provide a few images? How bad and what are the mitigations?